docs/advanced.md

```diff
@@ -374,16 +374,16 @@ HTTPX also allows you to specify the timeout behavior in more fine grained detai
 There are four different types of timeouts that may occur. These are **connect**,
 **read**, **write**, and **pool** timeouts.
 
-* The **connect timeout** specifies the maximum amount of time to wait until
+* The **connect** timeout specifies the maximum amount of time to wait until
```

Comment:
This change in emphasis is so that it's clearer now that the usage is `connect=...`. :-)

Reply:
Nicely done, yes.